handlers/vk/vk.go

```diff
@@ -417,6 +445,58 @@ func (h *handler) SendMsg(ctx context.Context, msg courier.Msg) (courier.MsgStat
 	return status, nil
 }
 
+// ToJSON returns the JSON encoding of Keyboard.
+func ToJSON(keyboard Keyboard) string {
+	b, _ := json.Marshal(keyboard)
+	return string(b)
+}
+
+//keyboard
+func buildQuickRepliesParams(msg courier.Msg) Keyboard {
+
+	var keyboard Keyboard
+	keyboard = AddRowKeyboard(keyboard)
+
+	for _, qr := range msg.QuickReplies() {
+		keyboard = AddTextButton(qr, qr, "secondary", keyboard)
+	}
```

Comment:
maybe we can do something similar to the telegram handler and use `utils.StringsToRows` to create better layouts when things don't fit on one row?